activiti-cloud-acceptance-scenarios/runtime-acceptance-tests/src/main/resources/stories/runtime-bundle/process-instance-actions.story

```diff
@@ -72,6 +72,9 @@ And a variable was created with name test_json_variable_result
 And a variable was created with name test_long_json_variable_result
 And a variable was created with name test_int_variable_result
 And a variable was created with name test_bool_variable_result
+And a variable was created with name test_long_variable_result
+And a variable was created with name test_bigdecimal_variable_result
```

Comment:
Here, I think we should check the variable type as well, not only the name.

`And a variable was created with name test_bigdecimal_variable_result and type bigDecimal`

Reply:
Makes sense. I will do it.

Reply:
@erdemedeiros I added check for big decimal variable result value, but not type. The type name for BigDecimal type variable stored in Query and Audit is set to `json`. I think we should have another issue to fix it in activiti-cloud project.

Reply:
It looks good to me. The issue related to type can be handled in another ticket.